### DISCUSSION TITLE
Show signatures in Git log

### DISCUSSION THREAD

**Discussion by leana8959** [PatrickF1/fzf.fish#296] [Q&A] [answered]
Hello,

I figured that it would be nice to be able to use the `--show-signature` option in the preview window so that we could see which commit is signed and which isn't. 

Thanks :)

**Comment by PatrickF1** [comment, marked as answer]
Hi Leana, can you do `showSignature = true` in the repo's git settings like this person did here? https://github.com/PatrickF1/fzf.fish/issues/286
Or making use of https://github.com/PatrickF1/fzf.fish#change-the-commit-formatting-used-by-search-git-log?

**Reply by leana8959** [reply]
Thank you! I didn't know this option existed!

**Reply by PatrickF1** [reply]
You're welcome! Leana, would you mind marking my answer as the answer?

**Reply by leana8959** [reply]
I noticed that when I do that, `fugitive` in neovim shows me the signature for each commit too, which is really redundant. But beside that, this is a perfect solution. :)
